testutil/testutil.go

```diff
@@ -300,7 +300,8 @@ func init() {
 					},
 				},
 				Resolve: func(p graphql.ResolveParams) (interface{}, error) {
-					return GetHuman(p.Args["id"].(int)), nil
+					id, _ := strconv.Atoi(p.Args["id"].(string))
```

Comment:
Since `Resolve`'s signature already returns an error, why suppress the error returned from `strconv.Atoi` ?